README.md

```diff
@@ -315,6 +319,70 @@ To update your Global Response Set, add one or more rows to the spreadsheet. To
 
 Caveat: deleting a response, and then re-adding the same response later will result in iAuditor Analytics dashboard treating these as different responses. This is because the new response will have a different internal identifier than the deleted response had. To update a response while keeping the same internal identifier you will need to use the response set API directly, instead of this tool. See the iAuditor developer portal for more details.
 
+### The Export Users tool
+
+This tool exports a list of users and their groups from iAuditor into a CSV file named `iauditor_users.csv`.
+
+To run it:
+Open a command-line terminal and navigate to the directory called `safetyculture-sdk-python/tools/export_users`.
+Run the following command:
+```
+python export_users.py --token <YOUR_IAUDITOR_API_TOKEN>
+```
+The exported CSV file will be saved in the current working directory.
+If the file already exists, it is overwritten.
+The exported CSV file columns contain the following user information and structure:
+- email
+- lastname
+- firstname
+- groups
+
+
+| email| lastname| firstname| groups|
+|---|---|---|---|
+|johnsmith@example.com |Smith|John|Group 1, Group 2|
+|johndoe@example.com|Doe|John|Group 3|
+|jasonR@example.com|R|Jason| |
+
+The field **groups** contains a comma-separated list of all iAuditor groups the user is a member of. All fields are string values.
+Note: comma-separated group names are not supported. If group names contains commas, it will be listed as is.
+
+### The Sync Users tool
+
+This tool updates the organisation users in iAuditor from a CSV file. After running this tool, users in iAuditor and their corresponding groups will match the contents of the CSV file.
```

Comment:
Please remove the line breaks after these sentences. Make it all into a paragraph.